### ISSUE TITLE
FileNotFoundException

### ISSUE THREAD

**Issue by TheIrishStud** [blahblahbal/Blah-s-MCreator-Mod#7]
molten_ice.json and mineral_water.json as well as ghast_tears.json created a FileNotFoundException in the latest log filing. Missing model for variant levels 0-15. "blahmod:blockstates/mineral_water.jason": java.io.FileNotFoundException : blahmod:blockstates/mineral_water.json and the same with the other 2. Also, if I am correct the enchantment descriptions mod is missing the description for repelling, molten_touch, xp_boost, stepping, and telekinesis. Finally, if I am also correct again, BiomeGenerationSettings is saying some features are "Not a JSON object" but some of those features don't look like anything from the "blahmod" so take that info with a grain of salt. Assuming this is all one needs to troubleshoot here and there I stand idle until I get a reply from you requesting further information. I could be wrong about this last part I found, just in case, but if I have read the debug log correctly following the failed loading of "blahmod:webstone" OpenGL error: 1281 (Invalid value), at: Expand/Copy VBO spams in the chat, causes heavy chunk loading lag, and frame drops all the way down to 0 fps.

**Comment by TheIrishStud** [comment]
I don't think it was this causing the 0 fps drops. Twas still experiencing heavy chunk loading lag after removing this from my mod list. I'm pretty sure I fixed my 0 fps drops though by removing a handful of other mods from my mod list so after trial and error I'll return with an update in case it's one of my favorites like this is if some compatibility update suggestions are required.

**Comment by TheIrishStud** [comment]
Confirmed Ascension Core is not the cause of 0 fps drops and chunk loading lag. However I found a bug or incompatibility issue if it can be called that but the Cedar trees cannot be vein mined using an axe and the "Reap mod" which allows you to chop a ground level log and chop the entire tree down. Cedar trees are also buried 1 or 2 block into the ground level with at least half the trees not buried floating with no dirt blocks underneath them. Can't say about the other biomes yet because half the time I'm just farming materials to make sure everything in my mod list works post updates. Apologies if I should've started a new issue instead rather than cluttering up this one.

**Comment by TheIrishStud** [comment]
I return with a sad face in tow... Twas searching for a Gem Land spawn but all I could find was Cedar Grove spawns following what I can only assume is the same webstone generation error which wasn't causing lag or fps drops before but I wasn't trying to intentionally spawn in one of Ascension Cores new biomes.

Tried to load a DUMMY block entity @ BlockPos{x=-22, y=17, z=113} but found not block entity block Block{blahmod:webstone} at location

**Comment by blahblahbal** [comment]
Hmm... interesting. In regards to the floating trees - this is something I have no clue how to fix as of now, since the way MCreator allows you to do trees (as structures) is kinda jank tbh. I'll eventually be switching off MCreator and remaking the entire mod, which will likely fix a lot of these issues.

Also very strange that Cedar trees can't be treecapitated with an axe when that mod is installed. I've been able to veinmine/treecapitate them using other mods, so maybe it's a bug with that Reap mod, dunno.

Further, in regards to the File not found thing - That's another issue with MCreator, and since it doesn't affect anything in-game (at least from what I can tell), I haven't bothered to try to fix it. It's likely just a superficial "bug" or whatever that doesn't actually do anything.

Sorry it took me so long to respond, I was super busy with other things and was taking a break from modding haha.
